LaCE-onboarding-guide.md

```diff
@@ -123,6 +123,9 @@ This page will help you solve common queries that you may face while working on
 
 ![Dummy-Math-Classroom](images/LaCEOnboardingGuide/Dummy-Math-Classroom.png)
 
+*Note*: If we face the error `Server error: 'None Type' object has no attribute 'version'`, go to `/release-coordinator page`(the user should have release-coordinator role), and under the MISC tab, click on `Flush Cache` button.
+
+![Flush-Cache-Button](images/LaCEOnboardingGuide/Flush-Cache-Button.png)
```

Comment:
In the screenshot, could you draw a red circle around the "Flush Cache" button? Thanks!

Reply:
PTAL @seanlip